packages/serverpod/lib/src/relic/web_server.dart

```diff
@@ -184,9 +184,9 @@ class WebServer {
   }
 
   /// Stops the webserver.
-  void stop() {
+  Future<void> stop() async {
     if (_httpServer != null) {
-      _httpServer!.close();
+      await _httpServer!.close();
```

Comment:
Suggestion: Change this to get a local variable so that we can remove the `!` cast.